src/test/resources/alma/HT012734833.json

```diff
@@ -6,25 +6,108 @@
   "title" : "Behavioural pharmacology",
   "hasItem" : [ {
     "type" : [ "Item" ],
+    "label" : "lobid Bestandsressource",
     "heldBy" : {
       "id" : "http://lobid.org/organisations/DE-605#!",
       "label" : "lobid Organisation"
     },
     "id" : "https://lobid.org/item/990108873860206441"
   }, {
     "type" : [ "Item" ],
+    "label" : "lobid Bestandsressource",
     "heldBy" : {
       "id" : "http://lobid.org/organisations/DE-A96#!",
       "label" : "lobid Organisation"
     },
     "id" : "https://lobid.org/item/991002245659706444"
   }, {
     "type" : [ "Item" ],
+    "label" : "lobid Bestandsressource",
     "heldBy" : {
       "id" : "http://lobid.org/organisations/DE-290#!",
       "label" : "lobid Organisation"
     },
     "id" : "https://lobid.org/item/991000300959706445"
+  }, {
+    "id" : "https://lobid.org/item/5333897520006444",
+    "type" : [ "Item" ],
+    "label" : "Portfolio",
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-632#!",
```

Comment:
Hm this denotes ExLibris. I think we should denote the library who "holds" it. Otherwise, how could we query e.g. "give me all Portfolios of DE-290" ?

Reply:
To be honest, i do not know what is wrong with the lookup here. There is no map to DE-632 in src/main/resources/alma/maps/alma-institution-code-to-isil.xml . And only a reference of it in line 65 :    ` morphVariables.put("isil", "DE-632"); ` of src/test/java/org/lobid/resources/AlmaMarc21XmlToLobidJsonTest.java.

Reply:
Right! It's the latter making the trouble. It is a fallback, hitting if an isil could not be found in the isil-mapping-xml.
There was an error in the morph: a regex group was incorrectly referenced with `{1}`. Correct is `${1}`. Fixed that with https://github.com/hbz/lobid-resources/pull/1121/commits/a2cbab9675590974cfbf2182ea6971f0a6001af3.